### PR TITLE
Add ECHO to makefiles for real silence

### PR DESCRIPTION
**Pull request by richsalz**
I know I could not remove the echo silliness, but at least now it's possible to turn it off.

### CODE REVIEW

File: Configurations/unix-Makefile.tmpl [hunk 1]
```diff
@@ -939,6 +940,7 @@ EOF
 # {libname}$shlibextsimple for the Unix platforms.
 $target: $lib$libext $deps $ordinalsfile
 	\$(MAKE) -f \$(SRCDIR)/Makefile.shared -e \\
+                ECHO=\$(ECHO) \\
```
Comment (levitte) [inline Configurations/unix-Makefile.tmpl]: indent with tabs here, just like the lines below this one.

**Comment by richsalz** [comment]
The whole file was rife with spaces ☺  fixed.

**Comment by levitte** [comment]
...  @richsalz, I commented on *one line*, not the whole file.  I was not at all complaining about the embedded perl sections.

The only reason to retain tabs in the actual Makefile parts is that `make` requires action lines to be indented with a tab.  Had it not been for that, I would have removed those tabs a long time ago.

**Comment by richsalz** [comment]
New commits pushed.  Will be squashed.

**Comment by richsalz** [comment]
thanks for the review!

**Pull request closed without merging**